docs/site/Appsody-LoopBack.md

```diff
@@ -0,0 +1,1018 @@
+    [Container] Wait received error on APPSODY_RUN/DEBUG/TEST signal: interrupt
+    [Container]
+    [Container] > nodejs-loopback-scaffold@1.0.0 pretest /project/user-app
+    [Container] > npm run clean && npm run build
+    [Container]
+    [Container]
+    [Container] > nodejs-loopback-scaffold@1.0.0 clean /project/user-app
+    [Container] > lb-clean dist *.tsbuildinfo
+    [Container]
+    [Container]
+    [Container] > nodejs-loopback-scaffold@1.0.0 build /project/user-app
+    [Container] > lb-tsc
+    [Container] > nodejs-loopback@0.1.7 start /project
+    [Container] > node -r source-map-support/register .
+    [Container]
+    [Container] Server is running at http://[::1]:3000
+    [Container] Try http://[::1]:3000/ping
+   ```
+
+7. In the `'API Explorer'`, try the `'GET /ping'` endpoint again, and you will
+   receive the new **greeting** response:
+
+   ```json
+   {
+     "greeting": "Hello from LoopBack 4!",
+     "date": "2019-11-21T18:22:22.975Z",
+     "url": "/ping",
+     "headers": {
+       "host": "[::1]:3000",
+       "connection": "keep-alive",
+       "accept": "application/json",
+       "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.97 Safari/537.36",
+       "sec-fetch-site": "same-origin",
+       "sec-fetch-mode": "cors",
+       "referer": "http://[::1]:3000/explorer/",
+       "accept-encoding": "gzip, deflate, br",
+       "accept-language": "en-US,en;q=0.9"
+     }
+   }
+   ```
+
+8. In the `'API Explorer'`, try the `'GET /todos'` endpoint.
+
+   Click on the `'Try it Out'` button, clear the filter field, and then click on
+   the `'Execute'` button.
+
+   ![lb4appsody_template_webpage_3.png](./imgs/lb4appsody_template_webpage_3.png)
+
+   The response will be:
+
+   ```json
+   [
+     {
+       "id": 1,
+       "title": "Take over the galaxy",
+       "desc": "MWAHAHAHAHAHAHAHAHAHAHAHAHAMWAHAHAHAHAHAHAHAHAHAHAHAHA"
+     },
+     {
+       "id": 2,
+       "title": "destroy alderaan",
+       "desc": "Make sure there are no survivors left!"
+     },
+     {
+       "id": 3,
+       "title": "play space invaders",
+       "desc": "Become the very best!"
+     },
+     {
+       "id": 4,
+       "title": "crush rebel scum",
+       "desc": "Every.Last.One."
+     }
+   ]
+   ```
+
+9. To see the docker container the Appsody started for you,
+   `open a new terminal` and use:
+
+   ```
+   appsody ps
+   ```
+
+   It will show something similar to this:
+
+   ```
+   CONTAINER ID    NAME                    IMAGE                           STATUS
+   d39c90c5191d    appsodylb4todo-dev      appsody/nodejs-loopback:0.1     Up About a minute
+   ```
+
+   `'docker ps'` will accomplish the same thing.
+
+### Stopping a LoopBack 4 Application Locally using Appsody
+
+If your application is currently running under Appsody control, you can stop it
+with `Ctrl+C`, or by running the command `'appsody stop'` from another terminal.
```

Comment:
what if I have multiple apps started by `appsody run`, would this command stop all of them? Is it possible to provide a container name as argument to specify which one to stop?

Reply:
> what if I have multiple apps started by `appsody run`, would this command stop all of them? Is it possible to provide a container name as argument to specify which one to stop?

Good question. But out of scope for this article. More for Appsody FAQ.

But from what I understand and observe, the `.appsody-config.yaml` file in your project directory (e.g. appsodyLB4Todo) has a `project-name` value:

```
project-name: appsodylb4todo
stack: appsody/nodejs-loopback:0.1
```

When `appsody run` is executed (from inside directory appsodylb4todo), it starts a docker container named 

```
appsodylb4todo-dev
```

To do: `appsody stop`, you must also be inside the directory appsodylb4todo .

Then it stops and removes this container.

So, if you have multiple apps started from separate project directories, and they have unique project directory names, you should be fine.

Reply:
> to do: appsody stop, you must also be inside the directory appsodylb4todo .

Magic! Now I understand how the run and stop cmd finds the app! thanks for explanation!